lib/b_inputoutput.dart

```diff
@@ -9,4 +9,11 @@
 
 // hint: gunakan stdin.readLineSync() untuk membaca input dari user
 
-void main() {}
+import 'dart:io';
+
+void main() {
+  String? nama = stdin.readLineSync();
+  int? umur = int.parse(stdin.readLineSync()!);
+
+  print("Halo, saya ${nama}, umur saya ${umur} tahun");
```

Comment:
Untuk string interpolation sederhana, hindari penggunaan kurung kurawal. Untuk lebih jelasnya dapat kamu baca di sini:

https://dart.dev/guides/language/effective-dart/usage#avoid-using-curly-braces-in-interpolation-when-not-needed